docs/source/basics/parameter_types.md

```diff
@@ -11,18 +11,36 @@ registration.
 
 ## Supported Types
 
-Below is an exhaustive list of supported types currently shipped with the SDK.
+Tasks support all native Python types, while workflow only supports a subset of types.
+
+Below is an exhaustive list of supported workflow parameter types currently shipped with the SDK, and how they will translate to the workflow GUI once the workflow is registered.
```

Comment:
not exhaustive:
- missing `dataclass`
- missing `Union`
- lists can contain anything inside
- should mention custom type transformers (just a link to flyte docs probably)

Reply:
Are you referring to this: https://docs.flyte.org/projects/cookbook/en/latest/auto/core/extend_flyte/custom_types.html